test/unit/three.source.unit.js

```diff
@@ -284,3 +284,6 @@ import './src/textures/FramebufferTexture.tests.js';
 import './src/textures/Source.tests.js';
 import './src/textures/Texture.tests.js';
 import './src/textures/VideoTexture.tests.js';
+
+//addons
+import './addons/utils/BufferGeometryUtils.tests.js';
```

Comment:
`three.source.unit.js` is only intended for `src` modules. We currently don't maintain unit tests for `examples/jsm`.

Reply:
Thanks for feedback. Do you want me to remove the test for acceptance ? I was thinking to do something to prevent eventual regression in the future.